templates/public_organization_search.html

```
@@ -0,0 +1,53 @@
+{% endblock %}
+{% block content %}
+<section class="apply-hero" aria-labelledby="organization-search-title" data-motion="hero">
+    <div class="apply-hero-content" data-motion-group="organization-search">
+        <div class="apply-intro" data-motion-child>
+            <h1 id="organization-search-title">Sök organisationsnummer</h1>
+            <p>Alla kan söka och se hur många aktiva privatkonton som är registrerade på ett organisationsnummer.</p>
+        </div>
+
+        <form method="get" class="apply-form" novalidate data-motion-child>
+            <label for="orgnr">Organisationsnummer</label>
+            <input
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)